packages/docs/src/pages/en/features/display-and-platform.md

```diff
@@ -0,0 +1,457 @@
+
+```html
+<script>
+  // Composables
+  import { useDisplay } from 'vuetify/lib/composables/display'
+
+  export default {
+    mounted () {
+      const display = useDisplay()
+
+      console.log(display.mobile.value) // false
+    }
+  }
+</script>
+```
+
+## Options
+
+The **display** composable has a numerous configuration options, such as the ability to define custom values for breakpoints.
+
+For example, the `thresholds` option modifies the values used for viewport calculations. The following snippet overrides *xs* through *lg* values, mobileBreakpoint, and increases `scrollbarWidth`.
+
+```js
+// src/plugins/vuetify.js
+
+import Vue from 'vue'
+import Vuetify from 'vuetify/lib'
+
+export default new Vuetify({
+  display: {
+    mobileBreakpoint: 'sm',
+    thresholds: {
+      xs: 340,
+      sm: 540,
+      md: 800,
+      lg: 1280,
+    },
+    scrollbarWidth: 24,
+  },
+})
+```
+
+<alert type="info">
+
+  The **mobileBreakpoint** option accepts numbers and breakpoint keys.
+
+</alert>
+
+## Examples
+
+In the following example, we use a switch statement and the current breakpoint name to modify the **height** property of the [v-card](/components/cards/) component:
+
+```html
+<!-- Vue Component -->
+
+<template>
+  <v-card :height="height">
+    ...
+  </v-card>
+</template>
+
+<script>
+  // Composables
+  import { useDisplay } from 'vuetify/lib/composables/display'
+
+  export default {
+    computed: {
+      height () {
+        const { name } = useDisplay()
+
+        // name is reactive and
+        // must use .value
+        switch (name.value) {
+          case 'xs': return 220
+          case 'sm': return 400
+          case 'md': return 500
+          case 'lg': return 600
+          case 'xl': return 800
+        }
+      },
+    },
+  }
+</script>
+```
+
+### Interface
+
+```ts
+{
+  // Breakpoints
+  xs: boolean
+  sm: boolean
+  md: boolean
+  lg: boolean
+  xl: boolean
+  smAndDown: boolean
+  smAndUp: boolean
+  mdAndDown: boolean
+  mdAndUp: boolean
+  lgAndDown: boolean
+  lgAndUp: boolean
+
+  // true if screen width < mobileBreakpoint
+  mobile: boolean
+  mobileBreakpoint: number | 'xs' | 'sm' | 'md' | 'lg' | 'xl'
+
+  // Current breakpoint name (e.g. 'xs' | 'sm' | 'md' | 'lg' | 'xl')
+  name: string
+
+  // The current value of window.innerHeight and window.innerWidth
+  height: number
+  width: number
+
+  // Device userAgent information
+  platform: {
+    android: boolean
+    ios: boolean
+    cordova: boolean
+    electron: boolean
+    chrome: boolean
+    edge: boolean
+    firefox: boolean
+    opera: boolean
+    win: boolean
+    mac: boolean
+    linux: boolean
+    intersection: boolean
+    touch: boolean
+    ssr: boolean
+  }
+
+  // Browser scrollbar width
+  scrollbarWidth: number
+
+  // The values used to make Breakpoint calculations
+  thresholds: {
+    xs: number
+    sm: number
+    md: number
+    lg: number
+  }
+}
+```
+
+## API
```

Comment:
This section being moved to api generator?

Reply:
Oh I forgot about that. Will need some help implementing.